agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryClient.java

```diff
@@ -236,7 +236,7 @@ private BatchItemProcessor initBatchItemProcessor(
     if (tempDir == null) {
       telemetryPipelineListener =
           new DiagnosticTelemetryPipelineListener(
-              "Sending telemetry to the ingestion service", false, " (telemetry will be lost)");
+              "Sending telemetry to the ingestion service", true, " (telemetry will be lost)");
```

Comment:
this looks like a bug

Reply:
tempDir is null.. written to disk is not possible. retries will be false. i don't get it why it becomes true.

Reply:
2nd arg controls logging, not writing to disk